src/main/java/com/homework/SpringAppTest.java

```diff
@@ -0,0 +1,30 @@
+package com.homework;
+
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class SpringAppTest {
+    public static void main(String[] args) {
+        String configLocation;
+        ClassPathXmlApplicationContext context = new ClassPathXmlApplicationContext(
```

Comment:
Why not in one line?
 ClassPathXmlApplicationContext context = new ClassPathXmlApplicationContext( "applicationContext.xml" );